pkg/cmd/roachtest/tests/disk_stall.go

```diff
@@ -383,16 +383,46 @@ func (s *cgroupDiskStaller) device() (major, minor int) {
 	}
 }
 
+type throughput struct {
+	limited        bool
+	bytesPerSecond int
```

Comment:
Nit: Not sure if we have a style rule against it, but does this need to be an int? If it were just a string we could set it to bytes or "max" accordingly and not need this throughput struct or the int to string conversion logic.